solidity/how-to-setup-hardhat.txt

```diff
@@ -2,9 +2,9 @@
 npm init
```

Comment:
I forgot about this file lol. I think we can remove it completely. I was just using it because it used to be confusing to set up buidlr projects. Once it's set up, of course, you can just install it with `npm install`